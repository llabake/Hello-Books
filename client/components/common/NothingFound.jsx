import React from 'react';
import PropTypes from 'prop-types';

const NothingFound = ({ text, id }) => {
  return (
    <div
      className="card-panel row center-align"
      id={id}>
      <h4>
        {text}
      </h4>
    </div>
  )
};

NothingFound.propTypes = {
  id: PropTypes.string,
  text: PropTypes.string
};

export default NothingFound;