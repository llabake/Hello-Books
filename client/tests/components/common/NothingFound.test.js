import React from 'react';

import NothingFound from '../../../components/common/NothingFound';

const wrapper = shallow(<NothingFound />);

test('<NothingFound /> mounts correctly', () => {
  expect(wrapper).toMatchSnapshot();
})

