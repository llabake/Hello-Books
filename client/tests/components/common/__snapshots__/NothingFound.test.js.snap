// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NothingFound /> mounts correctly 1`] = `
<div
  className="card-panel row center-align"
>
  <h4 />
</div>
`;
